test/NetTopologySuite.Tests.NUnit/IO/GML3/GML3WriterTest.cs

```diff
@@ -103,7 +106,7 @@ private static XDocument ToGML3(Geometry geom)
             using var ms = new MemoryStream();
```

Comment:
Change signature to
```private static XDocument ToGML3(Geometry geom, bool writeSrsName = true)```

Duplicate at least one of the actual tests where `writeSrsName` is set to `false`